style.css

```diff
@@ -92,3 +95,205 @@ header {
   color: #505f79;
   font-size: 20px;
 }
+
+/* -----work section----- */
+
+.works {
+  margin: 40px 0;
+  padding: 20px;
+  display: grid;
+  grid-template-columns: auto;
+  gap: 20px;
+}
+
+.project-card {
+  border: 1px solid #ebebff;
+  border-radius: 10px;
+  width: 95%;
+  margin: 8px auto;
+  background-color: #fff;
+  padding: 10px;
+}
+
+.canopy {
+  color: #344563;
+  margin-left: 12px;
+}
+
+.project-img {
+  display: flex;
+  justify-content: center;
+}
+
+.project-card img {
+  margin: 10px 0;
+}
+
+.project-title {
+  display: flex;
+  justify-content: start;
+  margin: 10px;
+}
+
+.project-title h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+/* .project-card span, img {
+  margin-right: 10px;
+  display: flex;
+  line-height: 16px;
+  font-size: 13px;
+} */
+
+/* .info {
+  display: flex;
+  align-items: center;
+  /* margin: 0 5px; */
+
+/* justify-content: center; */
+
+/* border: 1px solid red;
+  padding-right: 10px;
+} */
+
+.info {
+  margin: 0 auto;
+
+  /* display: flex; */
+
+  /* align-items: center;  */
+
+  /* justify-content: center; */
+}
+
+.info img,
+span {
+  margin-bottom: 0;
+  margin-right: 5px;
+  color: #7a869a;
+  font-size: 13px;
+}
+
+/* .info li {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin: 0 10px;
+} */
+
+.info .color {
+  color: #344563;
+}
+
+/* .faint {
+  color: #7a869a;
+  border: 1px solid red;
+} */
+
+/* span { */
+
+/* background-color: #7a869a; */
+
+/* width: 8px; */
+
+/* height: 8px; */
+
+/* border-radius: 50 */
+
+.project-paragraph {
+  display: flex;
+  text-align: start;
+  margin: 10px;
+  width: 292px;
+}
+
+.project-paragraph p {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.technologies {
+  display: flex;
+  margin: 10px 10px;
+}
+
+.technologies li {
+  margin-right: 10px;
+  padding: 5px 8px;
+  background-color: #ebebff;
+  border-radius: 5px;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.project-btn {
+  margin: 10px 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  background-color: #fff;
+  padding: 10px;
+}
+
+/* ----------About Section--------- */
+
+.about {
+  padding: 20px 30px;
+  margin: 40px 0;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  height: 140vh;
+}
```

Comment:
- [ ] You did a great job styling these elements 👏  but in order to make it match the original design better, you should consider adding some margin between the elements. For instance, between the "Get my resume" button and "Languages" title, there is a bigger spacing between them, as you can see here:
![Screen Shot 2021-11-27 at 08 59 36](https://user-images.githubusercontent.com/66967274/143680445-8de02809-29d3-4af7-9513-992c4cdc0ca4.png)
- [ ] The same goes for your list of technologies.